models/users.go

```diff
@@ -22,6 +22,12 @@ type UsersList struct {
 	Count int          `json:"count"`
 }
 
+type ListUserGroups struct {
+	Groups    []*cognitoidentityprovider.GroupType `json:"Groups"`
+	NextToken *string                              `json:"NextToken"`
+	Count     int                                  `json:"Count"`
```

Comment:
I think our JSON standards say that we should use lowercase with underscores e.g:
```
`json:"groups"`
`json:"next_token"`
`json:"count"`
```